.github/workflows/cicd.yaml

```diff
@@ -141,6 +141,93 @@ jobs:
           check_name: Linux Test Report
           comment_title: Linux Test Report
 
+  macos-cpp:
+    runs-on: macos-latest
+    if: github.repository == '4paradigm/OpenMLDB'
+    container:
+      image: ghcr.io/4paradigm/hybridsql:latest
+    env:
+      OS: darwin
+      ARCH: x86_64
+      NPROC: 3
+      CTEST_PARALLEL_LEVEL: 3 # parallel test level for ctest (make test)
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: prepare release
+        run: |
+          VERSION="snapshot"
+          if [[ ${{ github.ref }} == "refs/tags/v"* ]]; then
+            VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+            VERSION=${VERSION#v}
+            bash steps/prepare_release.sh "$VERSION"
+          fi
+          echo "OPENMLDB_PREFIX=openmldb-$VERSION-macos" >> $GITHUB_ENV
+
+      - name: enable java on release
+        if: github.event_name == 'push'
+        run: |
+          echo "SQL_JAVASDK_ENABLE=ON" >> $GITHUB_ENV
+
+      - name: configure
+        env:
+          SQL_JAVASDK_ENABLE: ${{ env.SQL_JAVASDK_ENABLE }}
+        run: |
+          make configure CMAKE_INSTALL_PREFIX=${{ env.OPENMLDB_PREFIX }}
+
+      - name: build
+        run: |
+          make build
+          # GitHub runner disk space is limited
+          # delete thirdparty build directory($ROOT/.deps/) to save disk space
+          # make thirdpartybuild-clean
+
+      - name: run cpp
+        id: unittest
```

Comment:
I guess we can omit unit tests, as well as the tests below @dl239

Reply:
Sorry, some steps deleted by mistake, I will fix it